gh-util.user.js

```diff
@@ -6,7 +6,7 @@
 // @match        https://github.com/*/pulls*
 // @match        https://github.com/*/pull/*
 // @grant        none
-// @run-at       document-start
+// @run-at       document-end
```

Comment:
```suggestion
// @run-at       document-start
```
By using `document-start`, we can ensure the immediate loading of this script and minimize the delay for the display of the **Files** and **Comment** buttons.

To fix the issue **Uncaught TypeError: Cannot read properties of null (reading 'appendChild')**, we can add the following conditional check:

```
if (!headerActions) {
    return;
}
```